src/documents/InfusionEventSystem.md

```diff
@@ -210,18 +212,17 @@ Once an event firer is constructed, it can be called with the following methods
   </thead>
   <tr>
     <td><code>addListener</code></td>
-    <td><code>listener: Function,String,
-listener specification
-</code></td>
+    <td><code>(listener {Function|String}[, namespace {String}, <br/> priority {String|Number}])</code></td>
     <td>
       Registers the supplied listener with this firer. The listener represents a function of a particular signature which is determined between the firer and listener of an event.
       The <code>namespace</code> parameter is an optional <code>String</code> which defines a key representing a particular <em>function</em> of the listener.
-      At most one listener may be registered with a firer with a particular namespace. This is a similar system to that operated by the JQuery namespaced events system.
+      At most one listener may be actively registered with a firer with a particular namespace. If a further listener B is registered with the same namespace as a previous one A, A will be
+      pushed onto a hidden stack, from the top of which it will be recovered by a call to <code>removeListener</code> removing B. Priorities are discussed in more detail on their own page [Priorities](Priorities.md).
```

Comment:
This link works fine in the docpad generated markup. There are several oddities with the markup processing with github which mean that we cannot always depend on it - for example, it uses a different algorithm for generating anchor ids.